src/pages/Mail/index.jsx

```diff
@@ -1,7 +1,71 @@
+import { Box, Icon, Stack, Typography } from "@mui/material";
 import React from "react";
+import { LiaHomeSolid } from "react-icons/lia";
+import { MdKeyboardArrowRight } from "react-icons/md";
 
 const MailPage = () => {
-  return <div>MailPage</div>;
+  return (
+    <Box
+      sx={{
+        width: "100%",
+        display: "block",
+        boxSizing: "border-box",
+        marginLeft: "auto",
+        marginRight: "auto",
+      }}
+    >
+      <Stack
+        sx={{
+          marginTop: "20px",
+          marginBottom: "24px",
+          borderRadius: "8px",
+          overflow: "hidden",
+          boxShadow: "none",
+          width: "100%",
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+          justifyContent: "space-between",
+          padding: "16px",
+          backgroundColor: "white",
+        }}
+      >
+        <Typography
+          sx={{
+            fontSize: "1.25rem",
+            fontWeight: "500",
+            lineHeight: "1.167",
+            color: (theme) => theme.palette.primary.dark,
+          }}
+        >
+          Mail
+        </Typography>
+        <Box
```

Comment:
It is better to use <Stack diraction="row"  alignItems="center" spacing={3}> {} </Stack >.
but using Box is ok.